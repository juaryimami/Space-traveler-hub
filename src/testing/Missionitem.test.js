import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MissionItem from '../components/mission/Missionitem';
import store from '../redux/configureStore';

it('renders correctly', () => {
  const item = {
    mission_id: 1,
    mission_name: 'Thaicom',
    description: 'testing',
    reserved: false,
  };
  const tree = renderer.create(
    <Provider store={store}>
      <MissionItem item={item} />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
