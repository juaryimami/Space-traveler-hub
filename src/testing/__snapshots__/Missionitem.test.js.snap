// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<tr>
  <td>
    Thaicom
  </td>
  <td>
    testing
  </td>
  <td>
    <span
      className="badge bg-secondary"
    >
      Not A Member
    </span>
  </td>
  <td>
    <button
      className="btn btn-outline-secondary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Join Mission
    </button>
  </td>
</tr>
`;
